lib/testRunner.js

```diff
@@ -242,7 +243,7 @@ class TestRunner {
               }
             }
             window.layoutShifts.push(event);
-          } catch (err) {}
+          } catch (err) { }
```

Comment:
You might need to configure Prettier in your IDE and / or just run `npm run prettier` from command line.